src/RwaLiquidationOracle2.sol

```diff
@@ -1,280 +0,0 @@
-// Copyright (C) 2020, 2021 Lev Livnev <lev@liv.nev.org.uk>
```

Comment:
Maybe we should save this contract somewhere else, in case we need it in the future?